### PR TITLE
Default Views

### PR DESCRIPTION
**Pull request by bencgreenberg**
## Description

Station currently presents an older version of the Vonage API Developer views for the `products` layout view and for the default landing view.

This PR changes them to guide the user on how to create their own views in their own instance of Station.

Station also has views that are idiosyncratic for a specific instance of Station (i.e. Vonage API Developer) and in those cases, they are removed and moved to that instance:

* Join the Vonage API Developer Slack

Additionally, there are partial views that are no longer in use at all and have been left unintentionally in the codebase:

* `xwithy`
* `podcast`
* `community`
* `dotnet`

### CODE REVIEW

File: lib/nexmo_developer/app/views/static/landing.html.erb [hunk 1]
```diff
@@ -17,30 +17,30 @@
...
         </div>
       </div>
 
       <hr class="hr--tall">
 
       <div class="center">
-        <h3>Do you have a question?</h3>
-        <p class="p-large">We're always here to help. Check out our <a href="https://help.nexmo.com">support FAQs</a>, contact our super helpful <a href="https://help.nexmo.com/hc/en-us/requests/new">support team</a> or get in touch on <a href="http://stackoverflow.com/questions/tagged/nexmo">Stack Overflow</a>.<br />Your data will be treated in accordance with our <a href="https://www.nexmo.com/privacy-policy">Privacy Policy</a>, which sets out the rights you have in respect of your data.</p>
+        <h3>Modifying This Content</h3>
+        <p class="p-large">To modify this content you can copy this file to the `custom/views/static` folder in your documentation folder and modify that copy. More detailed instructions to create your own landing page can be found on the <a href="https://nexmo.github.io/station/How-To-Use#customization-options">Station documentation</a>.</p>
```
Comment (mheap) [inline lib/nexmo_developer/app/views/static/landing.html.erb:43]: How do they know what this file is if they're seeing a rendered page? Should it link to the view on GitHub?

Comment (bencgreenberg) [reply lib/nexmo_developer/app/views/static/landing.html.erb:43]: Ah, yes. We can do that.

File: lib/nexmo_developer/app/views/slack/join.html.erb [hunk 2]
```diff
@@ -1,25 +0,0 @@
-<center>
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/views/slack/join.html.erb:1]: @bencgreenberg we need this view for `/community/slack`

File: lib/nexmo_developer/app/views/static/podcast.html.erb [hunk 3]
```diff
@@ -1,40 +0,0 @@
-<div class="container">
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/views/static/podcast.html.erb:1]: I think we need this one as well, we have a controller action for it.

**Review by fabianrbz — CHANGES_REQUESTED** [review]
Left a few comments, but otherwise, it looks good

File: lib/nexmo_developer/app/views/slack/join.html.erb (hunk 2 above)
Comment (bencgreenberg) [reply lib/nexmo_developer/app/views/slack/join.html.erb:1]: It's been moved to the complementary PR in NDP https://github.com/Nexmo/nexmo-developer/pull/3477

File: lib/nexmo_developer/app/views/static/podcast.html.erb (hunk 3 above)
Comment (bencgreenberg) [reply lib/nexmo_developer/app/views/static/podcast.html.erb:1]: It's unlinked and not referenced anywhere, I believe it was used for an event pre-config driven landing pages days.

File: lib/nexmo_developer/app/views/slack/join.html.erb (hunk 2 above)
Comment (fabianrbz) [reply lib/nexmo_developer/app/views/slack/join.html.erb:1]: aha, I didn't see that 🤦

File: lib/nexmo_developer/app/views/static/podcast.html.erb (hunk 3 above)
Comment (fabianrbz) [reply lib/nexmo_developer/app/views/static/podcast.html.erb:1]: Looks like it was used for `/hansel` See [this commit](https://github.com/Nexmo/nexmo-developer/commit/dd13c32bd92fb2db2e7b62f358437c1d785980c7).

That url is now handled as a landing page. Can you remove the controller action?

Comment (bencgreenberg) [reply lib/nexmo_developer/app/views/static/podcast.html.erb:1]: Yes, good catch. Will do now.

Comment (bencgreenberg) [reply lib/nexmo_developer/app/views/static/podcast.html.erb:1]: Done ✅

**Review by fabianrbz — APPROVED** [review]

**Pull request merged**